source/features/tags-dropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'dom-chef';
```

Comment:
Should have a comment at the top here that succinctly describes the feature.

Reply:
Sure.